histomicstk/segmentation/label/compact.py

```diff
@@ -3,7 +3,7 @@
 import scipy.ndimage.morphology as mp
 
 
-def CompactLabel(Label, Compaction=3):
+def compact(Label, Compaction=3):
```

Comment:
We need a better name that resembles what it does internally. `erase_protrusions` or `remove_protrusions`?